### ISSUE TITLE
Will this model support road  segmentation?  

### ISSUE THREAD

**Issue by intvo** [chrispolo/Keypoints-of-humanpose-with-Mask-R-CNN#8]
I would like to train the model to get the road segmentation, is it possible on this code?

**Comment by manueldiaz96** [comment]
I think maybe other network architecture would suit better road segmentation tasks. [This Pytorch repository](https://github.com/meetshah1995/pytorch-semseg) has some interesting code, I've worked with it and it is really intuitive. [This is a Tensorflow repo for semantic segmentation](https://github.com/GeorgeSeif/Semantic-Segmentation-Suite) but it seems like it is deprecated.
Remember that MaskRCNN is an instance segmentation model, which means that it combines the object detection and semseg tasks to give a more accurate representation than traditional ways that only give bounding boxes, so it works for classes like people, cars, etc., but not for buildings and road.
